docs/site/home.js

```diff
@@ -20,6 +22,20 @@ angular
       templateUrl: 'site/home.html',
       controller: 'HomeCtrl'
     });
+
+    markdownConverterProvider.config({
+      extensions: [function () {
+        return [{
+          type: 'lang',
+          regex: '\\[([^\\]]+)\\]\\((\\/[^)]+)\\)',
```

Comment:
Crazy. Would writing this out as a real RegExp() be possible, then sending the stringified value here?